test/pack/index.browser.test.ts

```diff
@@ -24,6 +24,29 @@ describe('pack', () => {
         expect(carParts.length).to.eql(7)
       })
 
+      it('can pack with custom unixfs importer options', async () => {
+        const { root, out } = await pack({
+          input: [new Uint8Array([21, 31])],
+          blockstore: new Blockstore(),
+          unixfsImporterOptions: {
+            cidVersion: 1,
+            chunker: 'fixed',
+            maxChunkSize: 262144,
+            hasher: sha256,
+            rawLeaves: true,
+            wrapWithDirectory: false
+          }
+        })
```

Comment:
As a user I'd think of these as options to `pack` rather than `unixfsImporter`... so perhaps we should expose them as a flat object...

```suggestion
      it('can pack with custom unixfs importer options', async () => {
        const { root, out } = await pack({
          input: [new Uint8Array([21, 31])],
          blockstore: new Blockstore(),
          cidVersion: 1,
          chunker: 'fixed',
          maxChunkSize: 262144,
          hasher: sha256,
          rawLeaves: true,
          wrapWithDirectory: false
        })
```

feels nicer i think

Reply:
I agree, the problem is that UnixFsImporter allows a gigantic list of options: https://github.com/ipfs/js-ipfs-unixfs/blob/master/packages/ipfs-unixfs-importer/src/types.d.ts#L51

If we want to go with just this subset, I think we can do the change you suggest. What do you think?

Reply:
ok, we probably dont need to expose all of those options just yet.

I'd be happy with just exposing `wrapWithDirectory` and maybe `maxChunkSize` for now.

@alanshaw what you think?

Reply:
that seems reasonable. I will do that change